js-miniapp-bridge/bridge.js

```diff
@@ -0,0 +1,64 @@
+var messageQueue = []
+window.MiniAppBridge = {};
+var uniqueId = Math.random();
+var isPlatform = {
+    Android: function() {
+        return navigator.userAgent.match(/Android/i);
+    },
+    iOS: function() {
+        return navigator.userAgent.match(/iPhone|iPad|iPod/i);
+    }
+};
+
+/**
+ * Method to call the native interface methods for respective platforms
+ * such as iOS & Android
+ * @param  {[String]} action Action command/interface name that native side need to execute
+ * @param  {[Function]} onSuccess Success callback function
+ * @param  {[Function]} onError Error callback function
+ */
+MiniAppBridge.exec = function(action, onSuccess, onError) {
+    const callback = {}
+    callback.onSuccess = onSuccess;
+    callback.onError = onError;
+    callback.id = ++uniqueId;
+    messageQueue.unshift(callback)
+    if(isPlatform.iOS()){
+        webkit.messageHandlers.MiniAppiOS.postMessage(JSON.stringify({action: action, id: callback.id}));
+    } else {
+        window.MiniAppAndroid.getUniqueId(JSON.stringify({action: action, id: callback.id}))
```

Comment:
Thanks from our discussion. This can be similar to iOS:
`window.MiniAppAndroid.postMessage(JSON.stringify({action: action, id: callback.id}))`

Reply:
Done